README.rst

```diff
@@ -398,6 +398,35 @@ Keys to take special care are the ones needed to configure Kafka and advertised_
      - ``lowest``
      - Decides on what basis the Karapace cluster master is chosen (only relevant in a multi node setup)
 
+Uninstall
+=========
+
+To unistall Karapace from the system you can follow the instructions described below. We would love to hear your reasons for uninstalling though. Feel free to drop us a line letting us know what we can improve!
+
+Installed via Docker
+--------------------
+
+If you installed Karapace via Docker, you would need to first stop and remove the images like described:
+
+First obtain the container IDs related to Karapace, you should have one for the registry itself and another one for the rest interface.
+
+  docker ps | grep karapace
+
+Afther this, you can stop each of the conatainers with:
+
+  docker stop <CONTAINER_ID>
+
+If you don't need or want to have the Karapace images around you can now proceed to delete them using:
+
+  docker rm <CONTAINER_ID>
+
+Installed from Sources
+----------------------
+
+If you installed Karapace from the sources via ``pithon setup.py install``, it can be uninstalled with the following ``pip`` command:
+
+  pip uninstall karapace
```

Comment:
``pip uninstall karapace``

Reply:
is this the same? or do you mean as a code block?

Reply:
my bad, my window was too small to see the indent :facepalm: